src/main/java/frc/robot/utilities/pid/TrigonPIDController.java

```diff
@@ -27,6 +27,16 @@ public void setSetpoint(double setpoint) {
         setSetpoint(setpoint, false);
     }
 
+    @Override
+    public double getError() {
+        return getPositionError();
+    }
+
+    @Override
+    public double getDeltaError() {
+        return getVelocityError();
+    }
+
```

Comment:
remove and put this in PIDFMotor instead

Reply:
same as above